### PR TITLE
HW3

### PR DESCRIPTION
**Pull request by alch-0**
#89

### CODE REVIEW

File: HW3/Аlех_Сhaganоff/HW3/HW3/ConsoleDrawing.cs [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        {
+            bool exitConfirmed = false;
+
+            switch (command)
+            {
+                case nameof(settings.KeyDrawDot): draw += commands.DrawDot;  break;
+                case nameof(settings.KeyDrawHorizontalLine): draw += commands.DrawHorizontalLine;  break;
+                case nameof(settings.KeyDrawVerticalLine): draw += commands.DrawVerticalLine; break;
+                case nameof(settings.KeyDrawSnowFlake): draw += commands.DrawSnowFlake;  break;
+                case nameof(settings.KeyClear):
```
Comment (spore9) [inline HW3/Аlех_Сhaganоff/HW3/HW3/ConsoleDrawing.cs]: The clear method doesn't work right, it's clear only one layer, if you draw dot 5 times, you need 5 clears.

Comment (alch-0) [reply HW3/Аlех_Сhaganоff/HW3/HW3/ConsoleDrawing.cs]: Yes, I missed that, thanks.
Now fixed.

File: HW3/Аlех_Сhaganоff/HW3/HW3/ConsoleInput.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿namespace HW3
+{
+    public class ConsoleInput : IReadInputProvider
```
[the remark this answers is not in the document]
Comment (alch-0) [reply HW3/Аlех_Сhaganоff/HW3/HW3/ConsoleInput.cs]: Separate input and output providers offer more flexibility in tuning the application.
I can match any input with any output wihout having to write new classes.
So I stick to this solution.

**Comment by alch-0** [comment]
> You should use 'StyleCop' to make your code more readable or use comments in some places.

Denis told us to get both approvals first and only then use StyleCop (before merginh the branch), that's why I haven't used it yet.

**Pull request merged**